test.py

```diff
@@ -4,8 +4,10 @@
 import unittest
 
 class MathTestCase(unittest.TestCase):
-    def verify(self, mkd_name, html_name, enable_dollar_delimiter=False):
-        config = {'enable_dollar_delimiter': enable_dollar_delimiter}
+    def verify(self, mkd_name, html_name, enable_dollar_delimiter=False,
+            render_to_span=False):
```

Comment:
Just for consistency with enable_dollar_delimiter which was already there. Should render_to_span be used differently than enable_dollar_delimiter then?